packages/Search/examples/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -12,7 +12,6 @@
 #include <DTK_DistributedSearchTree.hpp>
 
 #include <Kokkos_DefaultNode.hpp>
-#include <Teuchos_CommandLineProcessor.hpp>
```

Comment:
Yeah I saw it when I added it to the new build system

Reply:
Not the only header I missed

Reply:
Not sure I want to merge this and deal with conflicts though.  Hold on for now please.

Reply:
The new build system? You already have that?

Reply:
OK I dealt with the conflicts and decided to piggybacked on your PR.